stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputViewStyle.kt

```diff
@@ -123,8 +141,37 @@ public data class MessageInputViewStyle(
     @ColorInt public val commandButtonRippleColor: Int?,
     @ColorInt public val attachmentButtonRippleColor: Int?,
     public val messageInputInputType: Int,
+    // Message reply customization, by default belongs to center content as well
+    @ColorInt public val messageReplyBackgroundColor: Int,
+    public val messageReplyTextStyleMine: TextStyle,
+    @ColorInt public val messageReplyMessageBackgroundStrokeColorMine: Int,
+    @Px public val messageReplyMessageBackgroundStrokeWidthMine: Float,
+    public val messageReplyTextStyleTheirs: TextStyle,
+    @ColorInt public val messageReplyMessageBackgroundStrokeColorTheirs: Int,
+    @Px public val messageReplyMessageBackgroundStrokeWidthTheirs: Float,
 ) {
 
+    /**
+     * Creates an instance of [MessageReplyStyle] from the parameters provided by [MessageInputViewStyle].
+
+     *
+     * @return an instance of [MessageReplyStyle].
+     */
+    public fun toMessageReplyStyle(): MessageReplyStyle = MessageReplyStyle(
+        messageBackgroundColorMine = messageReplyBackgroundColor,
+        messageBackgroundColorTheirs = messageReplyBackgroundColor,
+        linkBackgroundColorMine = messageReplyBackgroundColor,
+        linkBackgroundColorTheirs = messageReplyBackgroundColor,
+        textStyleMine = messageReplyTextStyleMine,
+        textStyleTheirs = messageReplyTextStyleTheirs,
+        linkStyleMine = messageReplyTextStyleMine,
```

Comment:
Links don't seem to render inside quotes so I didn't want to expose an option that does not have an effect.